### PR TITLE
Add Cheryl Sabella as Core-dev

### PR DESCRIPTION
**Pull request by matrixise**
@vstinner Could you check if this PR is correct for @csabella 

Thank you so much.

### CODE REVIEW

File: developers.rst [hunk 1]
```diff
@@ -25,6 +25,9 @@ transliteration too.
 Permissions History
 -------------------
 
+- Cheryl Sabella was given push privileges on February 19, 2019 by Brett Cannon,
+  on the recommendation of Victor Stinner.
```
Comment (willingc) [inline developers.rst]: It would be nice to add @terryjreedy as well to the recommendation.

Comment (vstinner) [reply developers.rst]: Ooops, sorry @terryjreedy: fixed in commit 8b85ad54f22d85e5fae30fd91e8eac75e6f433b4.

Comment (matrixise) [reply developers.rst]: really sorry @terryjreedy it's my fault :/ I was not sure about the owner of the mentor.

Comment (terryjreedy) [reply developers.rst]: Carol, thank you for noticing. Others, no apologies needed.  I just glad to finally have her fully onboard.

Comment (eamanu) [reply developers.rst]: Congrats Cheryl!

**Pull request merged**